test/subscription_map.cpp

```diff
@@ -156,7 +155,7 @@ BOOST_AUTO_TEST_CASE( multi_non_wc_crud ) {
     auto con1 = std::make_shared<endpoint_t>(1);
     auto con2 = std::make_shared<endpoint_t>(2);
 
-    auto insert_or_update =
+    auto insert =
```

Comment:
It shouldn't be replaced.
It is for multi_index container. It does actually insert or update.
See the implementation.

Reply:
i will fix this